control-plane/grpc/proto/v1/blockdevice/blockdevice.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+import "v1/misc/common.proto";
+
+package v1.blockdevice;
+
+message BlockDevice {
+  // entry in /dev associated with device
+  string devname = 1;
+  // currently "disk" or "partition"
```

Comment:
Nit: `currently` -> `current`?

Reply:
Done.

Reply:
Actually I think the previous was probably correct. It was saying currently the devtype is either "disk" or "partition"